docs/items/items.md

```diff
@@ -30,4 +30,33 @@ Registering an Item
 
 Items must be [registered][registering] to function.
 
-[registering]: ../concepts/registries.md#registering-things
+Coloring an Item
+----------------
+
+There are items Minecraft prefers to render by coloring one single texture. Leather helmets, spawn eggs, potions and other.
+
+### Item Color Handler
```

Comment:
Pluralize "Handler" to "Handlers" to match with the "Block Color Handlers" section in the other file.